translation/source/variantTitle.xml

```diff
@@ -40,4 +40,6 @@
   <string name="go19x19">Surround the largest area(s) of the 19 by 19 board with your stones to win</string>
   <string name="backgammon">Get all your pieces into your home and then bear them off first to win</string>
   <string name="nackgammon">Get all your pieces into your home and then bear them off first to win</string>
+  <string name="breakthroughtroyka">The first player who makes a queen wins</string>
```

Comment:
you don't actually promote so maybe the following it better:
The first player who gets their piece to the other side wins

Reply:
will go with "Reach opponents home rank to win" unless you disagree